data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,32 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
+  "origin": {
+    "date_human-readable": "mid-4th c. CE",
+    "date_normalized": "0325/0375",
+    "language": "Greek",
+    "place": {}
+  },
+  "genre": ["Theological Works", "Commentary, Biblical", "Commentary, Old Testament", "Commentary, Genesis"],
+  "authors": [
+    {
+      "id": "https://sinaimanuscripts.library.ucla.edu/person/55",
+      "role": "author"
+    }
+  ],
+  "associated_persons": [],
+  "authority_files": [
+    {
+      "id": "https://haf.vhmml.org/work/320756617003",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron"
```

Comment:
maybe just need an array of authority file IDs? (it may be good to have a title to pull in case an authority file lacks a reliable API)